### DISCUSSION TITLE
Issue with creating colors with EV3: "TypeError: cannot create 'Color' instances"

### DISCUSSION THREAD

**Discussion by techniclyrobotics** [pybricks/support#1207] [Q&A]
Hi, our team is trying to create a color to use for a square up script but im getting errors.

Error code
```console
Traceback (most recent call last):
  File "/home/robot/TR_OfficialMasterpieceSeason/main.py", line 21, in <module>
  File "/home/robot/TR_OfficialMasterpieceSeason/run2.py", line 17, in <module>
TypeError: cannot create 'Color' instances
----------
Exited with error code 1.
```

Here's our whole run:
https://pastebin.com/A2WtgiSr
also ignore the ' at the end of the run I put that in while pasting its not in our real run

**Comment by dlech** [comment]
This is a Pybricks v3 feature. It doesn't work in v2.
